end_to_end/2-lineage-train-assess-bias-tune-registry-e2e.ipynb

```diff
@@ -0,0 +1,1083 @@
+{
```

Comment:
<p>A remark here, what does "value = 0.398" mean, is it good or bad, what does this number say about the bias? My guess it says only 39.8% samples are fraud. Is there a link to the Clarify doc page that helps customers to interpret those numbers?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>The class imbalance is biased against females that comprise 39.8% of the dataset.</p>

Reply:
<p>great idea I will link to : https://pages.awscloud.com/rs/112-TZM-766/images/Fairness.Measures.for.Machine.Learning.in.Finance.pdf</p>

Reply:
<p>Added: In this example dataset, the data is biased against females with only 38.9% of the data samples from female customers. We will address this in the next notebook where we show how we mitigate this class imbalance bias. Although we are only addressing Class Imbalance as an exemplar of bias statistics, you can also take into consideration many other factors of bias. For more detail, see :&nbsp;<a href="https://pages.awscloud.com/rs/112-TZM-766/images/Fairness.Measures.for.Machine.Learning.in.Finance.pdf" target="_blank">Fairness Measures for Machine Learning in Finance</a></p><p> </p>